ui/nuxeo-slots.js

```diff
@@ -111,11 +111,21 @@ window.nuxeo.slots.setSharedModel = (model) => {
         /**
          * A unique name for this slot.
          */
-        slot: {
+        name: {
           type: String,
           observer: '_register',
         },
 
+        /**
+         * @deprecated use `name` instead
```

Comment:
nit: it's common practice to document the version something was deprecated in, ie `@deprecated since 3.0`

Reply:
Will do, thanks.